bang/templates/items/costumeItem.html

```diff
@@ -0,0 +1,19 @@
+{% load tools %}
+{% load i18n %}
```

Comment:
I don't think you need these 2!

It's also better to avoid using `load`, especially in a loop (which is the case for list view), because it's extremely slow. That's why I sometimes add translated sentences in the context, to avoid loading `i18n`, and it's much faster.